### PR TITLE
Connection notification only when connected to WiFi

### PR DESCRIPTION
**Pull request by free4murad**
Fixes issue #1028

Changes: Complete connection notification is shown only if device is connected to WiFi

### CODE REVIEW

**Comment by free4murad** [comment]
@mananwason @the-dagger @mariobehling 
please review the PR.

File: android/app/src/main/java/org/fossasia/openevent/utils/ShowNotificationSnackBar.java [hunk 1]
```diff
@@ -56,6 +57,11 @@ public void onClick(View v) {
 
 
     public void buildNotification(){
+        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        android.net.NetworkInfo isWifi = connectivityManager.getNetworkInfo(ConnectivityManager.TYPE_WIFI);
+        if (!isWifi.isConnectedOrConnecting()){
+            return;
```
Comment (Abhi2424shek) [inline android/app/src/main/java/org/fossasia/openevent/utils/ShowNotificationSnackBar.java]: Can you make it true and put the code to display it in the if conditional statement pls, using return helps but putting the code reduces a line :) and improves the code viewing standard, else good work :)

Comment (free4murad) [inline android/app/src/main/java/org/fossasia/openevent/utils/ShowNotificationSnackBar.java]: @Abhi2424shek 
made the changes.

**Comment by saketkumar95** [comment]
@free4murad Nice work. Thanks for working on it. :) 
@the-dagger @mananwason LGTM :)

**Pull request merged**